client/src/pages/Listings/Listings.tsx

```diff
@@ -1,14 +1,17 @@
-import { useState, SyntheticEvent } from 'react';
-import { Grid, Box, Typography, Button } from '@mui/material';
+import { useState, SyntheticEvent, useEffect } from 'react';
+import { Grid, Box, Typography, Button, CircularProgress } from '@mui/material';
 import { DateRange } from '@mui/lab/DateRangePicker';
 import Layout from '../Layout/Layout';
 import SearchLocation from './SearchLocation/SearchLocation';
 import SearchDateRange from './SearchDateRange/SearchDateRange';
 import SitterCard from './SitterCard/SitterCard';
-import { Sitter, sampleData } from '../Profile/ProfileDetail/sampleData';
+import { Sitter } from '../Profile/ProfileDetail/sampleData';
+import { getAllProfiles } from '../../helpers/APICalls/profiles';
+import { Profile } from '../../interface/Profile';
 import useStyles from './useStyles';
 
 const Listings = (): JSX.Element => {
+  const [profiles, setProfiles] = useState<Profile[] | undefined>();
```

Comment:
Wouldn't it be better to the have an [] as the default state? 

`const [profiles, setProfiles] = useState<Profile[]>([]);`

Reply:
Applied